forms-flow-web/src/apiManager/services/applicationServices.js

```diff
@@ -59,6 +60,7 @@ export const fetchApplicationsAndDrafts = ({
       .then(({ data }) => {
         if (data) {
           dispatch(setApplicationListCount(data.totalCount || 0));
+          dispatch(setSubmissionFormName(data?.formName || ""));
```

Comment:
what is the use of this?

Reply:
The form name is collected form the response as it was causing issues in the previously as the forms without submissions not giving the data including form name. It is added in the new payload of this api.